config/HOWTO.md

```diff
@@ -0,0 +1,71 @@
+# Installation with geOrchestra
+
+Configuration of the [datadir](https://github.com/georchestra/datadir) of geOrchestra
```

Comment:
Installing Maelstro requires several customizations of the geOrchestra instance [datadir](https://github.com/georchestra/datadir):
 * copying and customizing files from [this folder](https://github.com/georchestra/maelstro/tree/main/config) into a `maelstro` subfolder of the datadir
 * security-proxy or gateway routes
 
It also requires an additional role to be created.